modules/learn/pages/data/durability.adoc

```diff
@@ -58,258 +62,303 @@ The correspondences are as follows:
+Couchbase Server promotes the replica to active status.
+If the promoted replica node fails, the new data becomes temporarily unavailable.
+When the promoted replica node restarts, the new data becomes available again on disk.
 
 |===
 
 [#protection-guarantees-2-replicas]
 == Protection Guarantees: Two Replicas
 
-The durability protection guarantees for two replicas are identical to those described above, for xref:learn:data/durability.adoc#protection-guarantees-1-replica[One Replica].
-This is because _majority_ is `2` for both cases: see the table in xref:learn:data/durability.adoc#majority[Majority], above.
+The durability protection guarantees for two replicas match those for one replica.
+The majority is 2 for both cases.
+See the table in <<#majority>>.
+
+If you configure two replicas, set the maximum number of sequential automatic failovers to 1 without administrator intervention.
+This setting makes sure auto-failover does not conflict with the durable write's guaranteed protection.
+
+[[maintaining-durable-writes]]
+== Maintaining Durable Writes During Single Replica Failovers
+
+As described in <<#majority>>, a bucket with one replicas must meet a majority requirement of two nodes for a durable write to succeed.
+Two nodes must acknowledge writing the data to either memory or disk (depending on the durability level) before Couchbase Server considers the write successful.
+If one node fails over, durable writes fail until the failed node rejoins the cluster.
+
+In some cases, this restriction can be inconvenient.
+For example, suppose you want to perform an upgrade using the xref:install:upgrade-procedure-selection.adoc#using-graceful-failover[graceful failover followed by a delta recovery method] in a three-node cluster.
+If you have a bucket with a single replica, performing the graceful failover causes durable writes to fail.
+This is because some vBuckets only have a single copy available in the cluster during the failover, so the write cannot achieve majority.
+
+In cases where you want Couchbase Server to report durable writes succeeded even if the majority of nodes are unavailable, you can enable the `durabilityImpossibleFallback` setting for the bucket. 
+This setting has Couchbase Server report a success for a durable write, even if the majority of nodes are unavailable.
+
+include::partial$maintain-durability-warning.adoc[]
+
+When you enable `durabilityImpossibleFallback`, Couchbase Server reports a success for durable writes even if the majority of nodes are unavailable.
+The following table shows the effects of the setting on durable writes in a three-node cluster for a bucket with a single replica.
+
+|===
+| Situation | `durabilityImpossibleFallback` Disabled | `durabilityImpossibleFallback` Enabled
+
+| All nodes available
+| Durable writes succeed
+| Durable writes succeed
+
+| One node fails over
+| Durable writes fail due to lack of majority
+| Durable writes succeed
```

Comment:
'Durable writes "succeed" without replication' or similar, a new status code is returned